worker/workercmd/worker_linux.go

```diff
@@ -26,12 +26,12 @@ type GardenBackend struct {
 	UseHoudini    bool `long:"use-houdini"    description:"Use the insecure Houdini Garden backend."`
 	UseContainerd bool `long:"use-containerd" description:"Use the containerd backend. (experimental)"`
 
-	Bin        string    `long:"bin"        description:"Path to a garden backend executable (non-absolute names get resolved from $PATH)."`
-	Config     flag.File `long:"config"     description:"Path to a config file to use for the Garden backend. Guardian flags as env vars, e.g. 'CONCOURSE_GARDEN_FOO_BAR=a,b' for '--foo-bar a --foo-bar b'."`
-	DNSServers []string  `long:"dns-server" description:"DNS server IP address to use instead of automatically determined servers. Can be specified multiple times."`
-	DNS            DNSConfig     `group:"DNS Proxy Configuration" namespace:"dns-proxy"`
-
-	RequestTimeout time.Duration `long:"request-timeout" default:"5m" description:"How long to wait for requests to Garden to complete. 0 means no timeout."`
+	Bin                string        `long:"bin"        description:"Path to a garden backend executable (non-absolute names get resolved from $PATH)."`
+	Config             flag.File     `long:"config"     description:"Path to a config file to use for the Garden backend. Guardian flags as env vars, e.g. 'CONCOURSE_GARDEN_FOO_BAR=a,b' for '--foo-bar a --foo-bar b'."`
+	DNSServers         []string      `long:"dns-server" description:"DNS server IP address to use instead of automatically determined servers. Can be specified multiple times."`
+	DNS                DNSConfig     `group:"DNS Proxy Configuration" namespace:"dns-proxy"`
+	RestrictedNetworks []string      `long:"restricted-networks" description:"Network ranges to which traffic from containers will be restricted. Can be specified multiple times."`
```

Comment:
Maybe rename the flag so it's the singular form (matches `dns-server` above)

```suggestion
	RestrictedNetworks []string      `long:"restricted-network" description:"Network ranges to which traffic from containers will be restricted. Can be specified multiple times."`
```

Reply:
Also, was it a deliberate decision to change from guardian's `deny-network` flag? I'm guessing yes but I'm not sure where we landed on naming containerd flags yet

Worth noting that if you specify the old Guardian flag `CONCOURSE_GARDEN_DENY_NETWORK` with containerd enabled, it doesn't error - it's just a silent no-op (which seems dangerous - maybe this'll be addressed in the future "flags" epic)

Reply:
Good point. The flags epic should come up with proper naming, and take care of the error handling you mention. I'm going to switch this back to use `CONCOURSE_GARDEN_DENY_NETWORK` for now instead of introducing a new flag that might get changed.